docs/getting_started.md

```diff
@@ -36,6 +36,22 @@ mirror_openshift:
   password: "PASSWORD"
 ```
 
+#### vSphere UPI
+For vSphere UPI deployments, RHCOS templates must be present on Cluster in Datacenter
+
+Till OCP version 4.9, download RHCOS templates from [here](https://raw.githubusercontent.com/openshift/installer/release-4.9/data/data/rhcos.json)
+e.g: From above url, form the exact download path using baseURI + vmware (in images section )
+
+[link](https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases/rhcos-4.9/49.84.202110081407-0/x86_64/rhcos-49.84.202110081407-0-vmware.x86_64.ova) to download OCP 4.9 RHCOS template
+
+Fom OCP 4.10 version, download RHCOS templates from [here](https://github.com/openshift/installer/blob/release-4.10/data/data/coreos/rhcos.json)
```

Comment:
A minor typo here `Fom` instead of `For`

Reply:
done